js/ui/map.js

```diff
@@ -985,6 +985,21 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Returns a Boolean indicating whether the map is currently in a "moving" state.
+     * Returns `false` if zooming, rotating, pitching, and dragging are all inactive.
+     *
+     * @returns {boolean} A Boolean indicating whether the map is moving.
+     */
+    isMoving: function() {
+        if (this.zooming ||
+            this.rotating ||
+            this.pitching ||
+            this.dragPan._active ||
+            this.dragRotate._active) return true;
```

Comment:
Interesting that we need to mix the "interaction handler" layer of abstraction the "camera" layer of abstraction to compute `Map#isMoving`. Should `Map#isMoving` return `true` if the user starts a "drag rotate" interaction but doesn't actually rotate the map? Or if the user starts a "drag pan" interaction but doesn't pan the map?